### ISSUE TITLE
Stutter

### ISSUE THREAD

**Issue by ChassterTP** [ruffle-rs/ruffle#13243]
Is it normal to stutter or lag when loading or playing flash games?

**Comment by n0samu** [comment]
Yes, particularly on lower-end devices or more demanding games. But please make sure hardware acceleration is enabled in your browser - performance suffers greatly if it isn't. Here are instructions for that: https://help.showpad.com/hc/en-us/articles/5428699999634-Enable-hardware-acceleration-in-your-browser
